packages/snapshot-listener/src/index.ts

```diff
@@ -0,0 +1,62 @@
+import express, {Request, Response} from "express";
+import {SnapshotEvent} from "./types";
+import {
+  RascalPublications,
+  RabbitMQController,
+  getRabbitMQConfig,
+} from "common-common/src/rabbitmq";
+import {factory, formatFilename} from "common-common/src/logging";
+import {RABBITMQ_URI} from "./config";
+import fetchNewSnapshotProposal from "./utils/fetchSnapshot";
+import {DEFAULT_PORT} from "./config"
+
+
+const app = express();
+const port = process.env.PORT || DEFAULT_PORT;
+app.use(express.json());
+
+let controller: RabbitMQController;
+
+app.get("/", (req: Request, res: Response) => {
+  res.send("OK!");
+});
+
+app.post("/snapshot", async (req: Request, res: Response) => {
+  try {
+    const event: SnapshotEvent = req.body.event;
+    if (!event) {
+      res.status(500).send("Error sending snapshot event");
+    }
+
+    const parsedId = event.id.replace(/.*\//, "");
+    const eventType = event.event.split("/")[1];
+    const response = await fetchNewSnapshotProposal(parsedId);
+    event.id = parsedId;
+    event.title = response.data.proposal.title;
+    event.body = response.data.proposal.body;
+    event.choices = response.data.proposal.choices;
+    event.space = response.data.proposal.space;
+    event.start = response.data.proposal.start;
+    event.expire = response.data.proposal.end;
+
+    await controller.publish(event, RascalPublications.SnapshotListener);
+    res.status(200).send({message: "Snapshot event received", event});
+  } catch (err) {
+    console.log(err);
+    res.status(500).send("error: " + err);
+  }
+});
+
+app.listen(port, async () => {
+  const log = factory.getLogger(formatFilename(__filename));
+  log.info(`⚡️[server]: Server is running at https://localhost:${port}`);
```

Comment:
Can we say "Snapshot Listener" instead of "Server" given how many servers we have?